validators/validators.js

```diff
@@ -2,7 +2,7 @@ const specialCharactersRegexTest = /[~%^*_+\-./\\`;~{}[\]"<>]/
 const dateRegexTest = /^[0-9]{4}-[0-9]{2}-[0-9]{2}$/
 
 const mediaSpecialCharactersRegexTest = /[~%^#*+\./\\`;~{}[\]"<>]/ // Allows dashes
-const mediaSubfolderRegexText = /^(images|files|(images|files)\/[^~%^*+\.\\`;~{}[\]"<>]+)$/
+const mediaSubfolderRegexText = /^(images|files|(images|files)\/[^#?~%^*+\.\\`;~{}[\]"<>]+)$/
```

Comment:
could we add a brief explanation on what this regex does? it's pretty undecipherable to me .-.

Reply:
Good point, https://github.com/isomerpages/isomercms-backend/pull/337/commits/efee09e07b57ca30ca147bb0f334e478c4d26a41

Reply:
i'm assuming that the banned characters are this section of the regex: `[^#?~%^*+\.\\`;~{}[\]"<>]+`? also it seems like it allows `files/<subdir>` so perhaps we should also update the comment accordingly.

sidenote: do we allow `<images/files>/` (note the extra `/`)

Reply:
changed in https://github.com/isomerpages/isomercms-backend/pull/337/commits/e10437f12443782a5d800ca66cf7fc982aeb1cee

We disallow ending with a `/` - we use slashes to denote a new directory only